canopeum_backend/canopeum_backend/settings.py

```diff
@@ -171,20 +171,15 @@ def get_secret(key: str, default: str):
 # Database
 # https://docs.djangoproject.com/en/5.0/ref/settings/#databases
 
-# DATABASES = {
-#     "default": "mysql://{}:{}@{}:3308/{}".format(
-#         os.getenv("MYSQL_USER"),
-#         os.getenv("MYSQL_PASSWORD"),
-#         os.getenv("MYSQL_HOST"),
-#         os.getenv("MYSQL_DATABASE"),
-#     ),
-# }
 DATABASES = {
     "default": dj_database_url.parse(
         "mysql://canopeum_user:{}@{}:3308/canopeum_db".format(
             get_secret("MYSQL_PASSWORD_CANOPEUM", ""),
             os.getenv("MYSQL_HOST_CANOPEUM", "localhost"),
         ),
+        # "default": dj_database_url.parse(
+        #     f"mysql://canopeum_user:{get_secret("MYSQL_PASSWORD_CANOPEUM", "")}
+        # @{get_secret("MYSQL_HOST_CANOPEUM", "localhost")}:3306/canopeum_db",
```

Comment:
Are these changes intentional? Even uncommented this doesn't make much sense

Reply:
MAde it clearier